### ISSUE TITLE
the errors and warnings are not underlined  

### ISSUE THREAD

**Issue by oovaa** [redhat-developer/vscode-java#2720]
[provide a description of the issue]

##### Environment
- Operating System: linux (kubuntu 22.04)
- JDK version:openjdk 18.0.2-ea 2022-07-19
- Visual Studio Code version: 1.71.2
- Java extension version: Extension Pack for Java v0.25.3

##### Steps To Reproduce
1. open .java file in vscode 
2. make a syntax error or warning 

[Please attach a sample project reproducing the error]
[Please attach logs](https://github.com/redhat-developer/vscode-java/wiki/Troubleshooting#enable-logging)

##### Current Result
the errors and warnings are not underlined  
##### Expected Result
the errors and warnings are underlined  
##### Additional Informations 
screenshot  
![Screenshot_20221005_082641](https://user-images.githubusercontent.com/103980029/194014937-9ca54af2-a2ec-45fb-b77e-09cd4d4c3b1d.png)

**Comment by jdneo** [comment]
Is the extension activated? You can check that if there is a 👍 icon at the bottom right corner of the status bar.

**Comment by joe-mccann-dev** [comment]
![image](https://github.com/redhat-developer/vscode-java/assets/53409795/d4d9a7fb-6287-48a2-9650-d0c8e1c9fa5f)

I'm having the same issue. Code will not underline in working tab. If I go to integrated terminal, problem does show there, If I  `ctrl + click` problem reference it shows underlined error in a new tab but not my current tab, which is not what it used to be and is harder to work with.

I'm using WSL Extension on Windows 11. I don't have many other extensions installed, maybe 3 others, and I've tried disabling all of them.

Things I've tried:

- reinstalling java extension pack
- reinstalling VS Code, deleting all data and configuration files
- cleaning language workspace command
- disabling other extensions
- deleting `settings.json`
